usmqe_tests/ui/test_cluster.py

```diff
@@ -0,0 +1,275 @@
+import pytest
+
+from usmqe.api.tendrlapi import glusterapi
+from usmqe.gluster import gluster
+
+LOGGER = pytest.get_logger('ui_cluster_testing', module=True)
+
+
+@pytest.mark.author("ebondare@redhat.com")
+@pytest.mark.happypath
+@pytest.mark.testready
+def test_cluster_import(application, valid_session_credentials):
```

Comment:
@ebondare would it be possible to enhance all four import cluster tests (`test_cluster_import`, `test_cluster_import_naming`, `test_cluster_import_profiling_disabled`, `test_cluster_import_view_progress`) in the way, that it will initially check if the cluster is not already imported and prospectively unmanage it before importing it?
It will be slightly better for integration with the current CI workflow (we can run all the tests in a row or repeat one test multiple times...).